### PR TITLE
Connection Banner: Adding themes slide and enhancing visuals

### PR DESCRIPTION
**Pull request by jeffgolenski**
Adding a themes slide into the connection banner and enhancing the visuals by adding illustrations instead of the circular icons in the content area.

- [x] Add themes slide
- [x] Add new illustrations to content areas of all slides
- [x] Set up test to run against new connection banner ( @ebinnion )
- [x] Place this connection banner on `/wp-admin/index.php`
- [x] Review copy 

**To test:**
1) Sync up with the `enhance/connection-banner-slides` branch
2) Disconnect jetpack from wordpress.com and view the plugins list page inside your site. `/wp-admin/plugins.php`

**Preview of connection banner on /wp-admin/ dashboard**

connection-banner-on-dash

**Detailed view of slides**

screen shot 2017-08-16 at 3 31 38 pm
screen shot 2017-08-16 at 3 31 47 pm
screen shot 2017-08-16 at 3 31 55 pm
screen shot 2017-08-16 at 3 32 02 pm
screen shot 2017-08-16 at 3 32 10 pm
screen shot 2017-08-16 at 3 32 17 pm
screen shot 2017-08-16 at 3 32 25 pm

**Example of mobile view**

screen shot 2017-08-16 at 3 34 42 pm

### CODE REVIEW

**Comment by jeffgolenski** [comment]
@nicoleckohler I've posted some screenshots for your convenience. Just note that the only copy that's been changed/added is for the themes slides. I pulled that content directly from jetpack.com/themes for a starting point. Thanks!

**Comment by jeffgolenski** [comment]
@ebinnion I've gone ahead and added a slide for "Themes" and also added all the necessary illustrations to each of the slides as well. Since all the slides are laid out by number, and I snuck the themes slide in the middle, I just gave it the number `3a` for now. Feel free to organize it however makes sense for you.

In the PR above I've included a mockup for how the connection banner should appear on `/wp-admin/index.php` NOW, with that being said, I'm cool if you want to create a new PR for that after these changes are merged in. We need to keep in mind that the connection banner is going to replace the connection widget area—having both of those being displayed will be a bit redundant and I think overwhelming. So I'm thinking that the banner will be the only thing to display on `wp-admin/index.php` pre-connection, and once the user connects, then we'll display the widget. Make sense? thoughts?

cc @dereksmart @jessefriedman @richardmuscat

**Comment by ebinnion** [comment]
Is this meant to replace #7622 then? Because I had set-up the split testing on that PR already.

**Comment by jeffgolenski** [comment]
@ebinnion No they're two separate things. #7622 is testing a new banner, while this is a modification of the current banner to test on the /wp-admin/ dashboard. If we need to break the tests out over multiple releases, let me know.

**Comment by ebinnion** [comment]
Ahh, ok. Thanks for the explanation

**Comment by nicoleckohler** [comment]
Themes banner copy changes:

- Change "hides all the complexity away" to "reduces complexity"
- Add a comma after "showcase" and change the "and" to "or" (you can browse _or_ search)
- Remove the dash in "one-click," add a comma after, and remove the "and" (so it should be "Preview, install, and activate in one click, then use...")
- I'm not fond of "just as you need it to" but can't think of anything super compelling to replace it with. Maybe "perfect for your site"? ... but it's OK as-is :)

**Comment by jeffgolenski** [comment]
Thanks @nicoleckohler ! Done.

**Comment by dereksmart** [comment]
@jeffgolenski all of the svg's are different widths, resulting in some weird height behavior with the slides.  

This one looks good
screen shot 2017-08-23 at 12 45 52 pm

This one has too much space 
screen shot 2017-08-23 at 12 46 09 pm

etc...

**Comment by jeffgolenski** [comment]
@dereksmart Yes, that's intentional that they have different widths. They're different widths because they all have different dimensions (none of the height and width ratios are the same between illustrations), so in order to get them to visually appear the same size, the widths are slightly different. 

I'll make an adjustment to that new themes svg that was just swapped out though. stay tuned

**Comment by dereksmart** [comment]
@jeffgolenski ok! 

I was playing around with it, and if you remove the bottom margins from `.jp-wpcom-connect__content-icon`, all of the slides are the same height (except for the stats card), I think it's nicer to not have the heights jump around when clicking through.

**Comment by ebinnion** [comment]
Pushed a change so that we use the new `build_connect_url_for_slide()` method which will handle the `from` parameter, taking into account location of the banner, the banner version, and the slide number.

File: class.jetpack-connection-banner.php [hunk 1]
```diff
@@ -216,15 +222,8 @@ class="notice-dismiss connection-banner-dismiss"
...
-								</g>
-							</svg>
+						<div class="jp-wpcom-connect__content-icon jp-connect-illo">
+							<img src="<?php echo plugins_url( 'jetpack/images/jetpack-welcome.svg', dirname( __FILE__ ) ); ?>" alt="Jetpack is a free plugin that utilizes powerful WordPress.com servers to enhance your site and simplify managing it." height="auto" width="250" />
```
Comment (dereksmart) [inline class.jetpack-connection-banner.php]: This alt description should be translated

File: class.jetpack-connection-banner.php [hunk 2]
```diff
@@ -278,8 +277,8 @@ class="dops-button next-feature"
 					<div class="jp-wpcom-connect__slide jp-wpcom-connect__slide-two">
 						<h2><?php esc_html_e( 'Detailed stats and traffic tools to help your site grow', 'jetpack' ) ?></h2>
 
-						<div class="jp-wpcom-connect__content-icon">
-							<svg xmlns="http://www.w3.org/2000/svg" width="100" height="100" viewBox="687 83 100 100" version="1.1"><g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" transform="translate(687.000000, 83.000000)"><circle fill="#3D596D" cx="50" cy="50" r="50"/><path d="M45.5 63L55.5 63 55.5 28 45.5 28 45.5 63ZM60.5 63L70.5 63 70.5 35.5 60.5 35.5 60.5 63ZM30.5 63L40.5 63 40.5 45.5 30.5 45.5 30.5 63ZM28 73L73 73 73 68 28 68 28 73Z" fill="#FFFFFF"/></g></svg>
+						<div class="jp-wpcom-connect__content-icon jp-connect-illo">
+							<img src="<?php echo plugins_url( 'jetpack/images/stats-people.svg', dirname( __FILE__ ) ); ?>" alt="Get clear and concise stats and analytics about your visitors." height="auto" width="225" />
```
Comment (dereksmart) [inline class.jetpack-connection-banner.php]: alt description should be translated

File: class.jetpack-connection-banner.php [hunk 3]
```diff
@@ -323,8 +322,8 @@ class="dops-button next-feature"
...
+						<div class="jp-wpcom-connect__content-icon jp-connect-illo">
+							<img src="<?php echo plugins_url( 'jetpack/images/security.svg', dirname( __FILE__ ) ); ?>" alt="Your site is automatically protected from brute force attacks, plus you can use single sign-on for extra security." height="auto" width="250" />
```
Comment (dereksmart) [inline class.jetpack-connection-banner.php]: All the alt descriptions should be translated!

**Comment by jeffgolenski** [comment]
@dereksmart Can you sweep though again and verify those alt tags are ready for translation?

**Comment by jeffgolenski** [comment]
Thanks for the merge, @ebinnion !

**Pull request merged**